Package.swift

```diff
@@ -11,7 +11,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/vapor/fluent-kit.git", from: "1.0.0-beta.2"),
-        .package(url: "https://github.com/vapor/postgres-kit.git", from: "2.0.0-beta.2"),
+        .package(url: "https://github.com/skelpo/postgres-kit.git", .branch("master")),
```

Comment:
We need to make sure this gets changes back to vapor/postgres-kit before it gets merged and tagged.